### PR TITLE
First cut on Spring XD with other comparable products.

### PR DESCRIPTION
**Pull request by sabbyanandan**
First-cut on content for:
- Spark/Spark Streaming
- Storm
- Sqoop
- Flume
- Oozie

Format followed:
- Technology description
- Spring XD play 
- Bullets - value proposition
- Generalization / conclusion

### CODE REVIEW

File: spring-xd-related-work.tex [hunk 1]
```diff
@@ -1,2 +1,99 @@
...
+
+\subsection{Spring XD and Storm}
+Storm is a distributed computation system for real-time stream processing.
+
+For customers who are in need of real-time stream processing, distributed data computations, and ETL capabilities, Spring XD as an unified runtime supports variety of use-cases ranging from classic enterprise to Big Data and IoT. 
+
+Spring XD can do the following which Storm can't do.
+
+\begin{itemize}
+\item Spring XD’s Shell compared with Storm’s API programming-paradigm delivers immediate productivity. No coding, IDE, bundling or packaging necessary. The high-level DSL abstracts complexities through developer-friendly fixtures.
```
Comment (cppwfs) [inline spring-xd-related-work.tex]: The '’' is a control character and causing LaTex to not format it properly.  Change apostrophe in the XD’s to Spring XD's.   This is a carry over from copying a Google doc to a Latex.tex file.  Same with Storm’s, needs to be change to Storm's.

File: spring-xd-related-work.tex [hunk 2]
```diff
@@ -1,2 +1,99 @@
...
+\item REST-APIs allows topology be built in isolation without having to disrupt existing pipelines.
+\item Loosely coupled ‘modules’ that may be responsible for ingestion, analytics, data-processing, machine-learning or data-export can be individually managed and dynamically scaled.
+\item Simplified governance model for ‘modules’ (unit-of-work) and colocation capabilities.
+\item Composition of ‘modules’ (unit-of-work) improves performance characteristics.
```
Comment (cppwfs) [inline spring-xd-related-work.tex]: Apostrophe issues with ‘modules’.  Needs to be 'modules'.

File: spring-xd-related-work.tex [hunk 3]
```diff
@@ -1,2 +1,99 @@
...
+\begin{itemize}
+\item Nothing manual and no coding! The high-level DSL abstracts configuration and setup. One-line DSL declaration automates data collection, processing, and export to a desired format or destination.
+\item Manage data-centric workflows with REST-APIs either via DSL, admin-ui, or custom applications.
+\item Administer or monitor data pipelines through Jolokia or JMX endpoints. 
+\item >500 combinations of data pipelines/flows can be created OOTB.
+\item Granular control options to intercept step-execution of data pipeline to create complex data-driven workflows.
+\item Flexibility through ‘Deployment Manifest’ to declaratively configure data partitioning strategy to route data to a specific consumer instance in the cluster. Improves performance characteristics by virtue of colocation.
+\end{itemize}
+
+No more config files, IDE, bundling, packaging or deploying artifacts. Spring XD’s developer-friendly fixtures delivers productivity.
```
Comment (cppwfs) [inline spring-xd-related-work.tex]: Apostrophe fun.

File: spring-xd-related-work.tex [hunk 4]
```diff
@@ -1,2 +1,99 @@
...
+\item Ability to create and launch workflow-jobs from admin-ui. Historical snapshots of execution, errors and states are available for exploration via admin-ui.
+\item Portable runtime that can run where there is JVM. On-prem, Pivotal CloudFoundry, YARN, Mesos, Docker or EC2.
+\end{itemize}
+
+Bridging the gaps between offline and near-real-time data, Spring XD will continue to evolve as one-stop platform for data workflows to efficiently process the data in-transit, at-rest, and in-use.
+
+\subsection{Spring XD and Sqoop}
+Sqoop assists with data transmission between Hadoop and relational databases.
+
+For customers who need data ingest and export between various databases and Hadoop, Spring XD’s unified platform facilitates out-of-the-box fixtures to orchestrate data movements reliably.
```
Comment (cppwfs) [inline spring-xd-related-work.tex]: Apostrophe fun.

**Comment by pperalta** [comment]
Merged as 9927ea7b83920570362c1f0e5c3e4c76a939884c

**Pull request closed without merging**